haproxy_exporter.go

```diff
@@ -89,6 +89,7 @@ func (m metrics) String() string {
 
 var (
 	serverMetrics = metrics{
+		2:  newServerMetric("current_queue", "Current server queue length.", nil),
```

Comment:
It's not queueing servers but requests. Maybe let's improve this and the backend's help message. First idea:

server: Current number of queued requests assigned to this server.
backend: Current number of queued requests not assigned to any server.